workspaces/orchestrator/plugins/orchestrator/src/api/OrchestratorClient.ts

```diff
@@ -116,17 +118,18 @@ export class OrchestratorClient implements OrchestratorApi {
       await this.getDefaultReqConfig();
     const authTokens: { provider: string; token: string }[] = [];
 
+    const authRequestOptions: AuthRequestOptions = { optional: true };
     /** Build one promise per provider (guard against missing APIs) */
     const tokenPromises = [
       this.githubAuthApi
         ? this.githubAuthApi
-            .getAccessToken?.()
+            .getAccessToken?.(undefined, authRequestOptions)
```

Comment:
Doubt here.
According to the `getAccessToken` description "[....] The returned promise can be rejected, but only if the user rejects the login request." Do you think we need to add a `.catch(....)` to handle this scenario?
This comment applies to GitLab part as well

Reply:
With this change, the login form is never displayed, so the user can not reject it.

This code is kind of "collector" of tokens which are already in the cache.
We will add option to login into selected SCMs within FLPATH-2314.